assets/css/blog.css

```diff
@@ -23,9 +23,6 @@ iframe {
     margin: 0 auto;
     display: block;
     border: 1px solid #ccc;
-    seamless;
-    frameborder: 0;
-    scrolling: no;
```

Comment:
According to https://caniuse.com/ all these attributes are not supported by any browser any more